.github/workflows/tagging.yml

```diff
@@ -19,17 +19,17 @@ jobs:
           echo $commitmsg
       - name: get tag version
         run: |
-          echo ::set-env name=tagversion::$( sed -e 's#.*: \(\)#\1#' <<< ${commitmsg} )
-          echo $tagversion
+          echo ::set-env name=tagversion::$( egrep -o '([0-9]+\.){2}(\*|[0-9]+)(-.*)?' <<< ${commitmsg} )
```

Comment:
this seems like is should only pull the version number and ignore everything else. @nellyk can you list some scenarios this has been tested with

Reply:
could you add the `v` here instead of each time below ?

Reply:
yes this are some of the scenarios tested 

- invalid tag version pushed `5.0` https://github.com/nellyk/one-app/actions/runs/84730796
 - invalid tag version pushed `5.0.8` https://github.com/nellyk/one-app/actions/runs/84730639 
 -valid tag version pushed `v5.0.8` https://github.com/nellyk/one-app/actions/runs/84730349